index.html

```diff
@@ -105,7 +105,8 @@ <h2 id="document_purpose">Purpose of this document</h2>
   Note that the terminology is based on that defined in [[JLREQ]]. </p>
 
 <aside class="note" title="Ruby and interlinear notes" id="n20200529001"> Interlinear notes are sometimes placed between lines
-      in the same position as ruby annotations.
+      in the same position as ruby annotations
+      (see also JLReq <a href="https://www.w3.org/TR/jlreq/#fig3_2_8">Figure 241: An example of a note in inter lines</a> for examples).
```

Comment:
Sorry i didn't see this sooner. I think it's a bad idea to list 'Figure 241', since the figure number may change.  The caption for that figure is also very poor English (and is probably worth changing in JLREQ).  

I suggest to replace the text with the following:

> (see also <a href="https://www.w3.org/TR/jlreq/#fig3_2_8">an example of an interlinear note in JLReq </a>).